lib/repo/git_repository.rb

```diff
@@ -30,15 +30,15 @@ class GitRepository < Repository::AbstractRepository
     # created using GitRespository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
+      if IS_REPOSITORY_ADMIN.nil?
```

Comment:
Thanks.  I'm getting rid of Repository.con altogether in another commit, but I'll wait to completely pull it out until after we merge your tests.